scss/_variables.scss

```diff
@@ -1159,8 +1159,9 @@ $form-range-thumb-border-radius:           50% !default;
 $form-range-thumb-box-shadow:              0 .1rem .25rem rgba($black, .1) !default;
 $form-range-thumb-focus-box-shadow:        null !default;
 $form-range-thumb-focus-box-shadow-width:  $input-focus-width !default; // For focus box shadow issue in Edge
-$form-range-thumb-active-bg:               null !default;
-$form-range-thumb-active-border:           $component-active-bg !default; // Boosted mod
```

Comment:
We cannot remove the Sass variable without a transition period where we announce that it is deprecated.
You can do something like that:

```scss
// fusv-disable
$form-range-thumb-active-border:           $component-active-bg !default; // Boosted mod / Deprecated in v5.2.2
// fusv-enable
```

And maybe in the migration guide say that it is deprecated in v5.2.2 and will be removed in v5.3.0 🤷